### PR TITLE
Fix : 9784 Added the note for display.max_colwidth in dsintro.rst

### LINKED ISSUE #9784: DataFrame.to_string truncates long strings
I am calling to_string() without any parameters and it beautifully fixed-formatted my dataframe apart from my very wide filename column, that is being truncated with "...". How can I avoid that?

``` text
                                            FILENAME  OBS_ID  XUV  
0  'mvn_iuv_l1a_IPH3-cycle00007-mode040-muvdark_2...      40  MUV  
1  'mvn_iuv_l1a_IPH2-cycle00047-mode050-muvdark_2...      50  MUV  
2  'mvn_iuv_l1a_apoapse-orbit00127-mode2001-muvda...    2001  MUV  
3  'mvn_iuv_l1a_APP1-orbit00087-mode1031-fuvdark_...    1031  FUV  
4  'mvn_iuv_l1a_IPH2-cycle00005-mode060-fuvdark_2...      60  FUV  
```

I tried calling it like this, but to no avail (same output):

``` python
with open('test_summary_out.txt','w') as f:
    f.write(summarydf.head().to_string(formatters={'filename':lambda x: "{:100}".format(x)}))
```

Version: 0.16 with Python 3.4

### PR DESCRIPTION
**Pull request by prabhjotsumman**
- [x] closes #9784
- [ ] tests added / passed
- [ ] passes `git diff upstream/master | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/dsintro.rst [hunk 1]
```diff
@@ -720,6 +720,15 @@ option:
 
    pd.DataFrame(np.random.randn(3, 12))
 
+Note : You can also adjust the max width of the individual columns by using `display.max_colwidth`
+option:
+
+.. ipython:: python
+
+   pd.set_option('display.max_colwidth',500)
```
Comment (jreback) [inline doc/source/dsintro.rst]: use a context manager here, and show an example where this actually matters

**Comment by Prabhnith** [comment]
ok sir, i will add an example with Long names or strings that will make it more clear.

**Comment by Prabhnith** [comment]
sir, any reviews

File: doc/source/dsintro.rst [hunk 2]
```diff
@@ -720,6 +720,21 @@ option:
 
    pd.DataFrame(np.random.randn(3, 12))
 
+Note : You can also adjust the max width of the individual columns by using `display.max_colwidth`
+option:
+
+.. ipython:: python
+
+   datafile={'filename':['children.jpg','flower.jpg'],
+   	     'path':["media/prabhjot/PRABHJOT/WALLPAPERS/children.jpg",
```
Comment (jreback) [inline doc/source/dsintro.rst]: use a more generic filename here

**Comment by Prabhnith** [comment]
filename changed to generic

**Comment by jreback** [comment]
thanks!

**Pull request closed without merging**